console_backend/src/status_bar.rs

```diff
@@ -166,10 +166,12 @@ impl StatusBar {
         status_bar_status.set_ins(&sb_update.ins_status);
         status_bar_status.set_data_rate(sb_update.data_rate);
         status_bar_status.set_solid_connection(sb_update.solid_connection);
-        status_bar_status.set_title(&format!(
-            "{} - Swift Console {}",
-            sb_update.port, sb_update.version
-        ));
+        let port = if sb_update.port.is_empty() {
+            ""
+        } else {
+            sb_update.port + " - "
+        };
+        status_bar_status.set_title(&format!("{port}Swift Console {}", sb_update.version));
```

Comment:
For consistency, we probably shouldn't mix the two styles of format strings:

```suggestion
        status_bar_status.set_title(&format!("{}Swift Console {}", port, sb_update.version));
```